gear-programs/erc20-relay/app/src/service.rs

```diff
@@ -173,33 +138,7 @@ where
             _ => return Err(Error::InvalidReceiptProof),
         }
 
-        let call_payload = SubmitReceipt::encode_call(message.receipt_rlp);
-        let (vft_manager_id, reply_timeout, reply_deposit) = {
-            let state = self.state.borrow();
-
-            (
-                state.vft_manager,
-                state.config.reply_timeout,
-                state.config.reply_deposit,
-            )
-        };
-
-        gstd::msg::send_bytes_for_reply(vft_manager_id, call_payload, 0, reply_deposit)
-            .map_err(|_| Error::SendFailure)?
-            .up_to(Some(reply_timeout))
-            .map_err(|_| Error::ReplyTimeout)?
-            .handle_reply(move || handle_reply(slot, transaction_index))
-            .map_err(|_| Error::ReplyHook)?
-            .await
-            .map_err(|_| Error::ReplyFailure)?;
-
-        let _ = self.notify_on(Event::Relayed {
-            slot,
-            block_number: block.body.execution_payload.block_number,
-            transaction_index: transaction_index as u32,
-        });
-
-        Ok(())
+        Ok(message.receipt_rlp)
```

Comment:
place result value in to the match expression

```suggestion
        match trie.get(&key_db) {
            Ok(Some(found_value)) if found_value == value_db => Ok(message.receipt_rlp),
            _ => Err(Error::InvalidReceiptProof),
        }
```